x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -148,12 +152,12 @@ export const EnginesList: React.FC = () => {
               {i18n.translate(
                 'xpack.enterpriseSearch.content.engines.searchPlaceholder.description',
                 {
-                  defaultMessage: 'Locate an engine via name or indices',
+                  defaultMessage: "Locate an engine via name or by it's included indices indices.",
```

Comment:
Think you wrote indices twice here?

Reply:
Also should be "its" instead of "it's"

(driveby nitpick)

Reply:
I always get that wrong too.

Reply:
Fixed in latest commit. Not sure how I missed it !